### PR TITLE
Add Console access

### PR DESCRIPTION
**Pull request by tristanmorgan**
Hi, This is the resurrection of a PR for adding access to the AWS Console from the aws-creds util. (the old PR sat for over 6 months without a comment).

### CODE REVIEW

**Comment by tristanmorgan** [comment]
Hi @maneframe , just wondering if anyone is watching this project or are they just "moving to vaulted"?

**Comment by simonista** [comment]
Hey @tristanmorgan, sorry for the lack of communication before.  I assume you're talking about https://github.com/instructure/aws-keychain-util/pull/20 ?  Anyway, you're right that most people have moved over to vaulted, but we still have a few apps integrated into aws-keychain-util, because of how nice the integration with the macos keychain is.  Let me ping a few people internally, but if I'm reading the other PR correctly, sounds like we should be able to move forward with this.  I just want to spend a little time understanding all the little "extra" cleanup changes besides the core add of the console command.

File: bin/aws-creds [hunk 1]
```diff
@@ -164,15 +166,12 @@ when 'rm'
...
-  name = get_name_from_args_for_command('add-role')
-  item = get_valid_item(name)
-
-  role     = ask('         role name: ')
-  arn      = ask('          role arn: ')
+  role_name = ask('         role name: ')
+  arn       = ask('          role arn: ')
 
   keychain.generic_passwords.create(
-    label: "#{name} role #{role}",
```
Comment (tristanmorgan) [inline bin/aws-creds:174]: The (account) name isn't needed and prevents cross account roles.

**Comment by tristanmorgan** [comment]
Hi @simonista was there anything not clear/need me to change?

File: bin/aws-creds (hunk 1 above)
Comment (simonista) [reply bin/aws-creds:174]: If I'm understanding this correctly, I have two concerns:

1. we're changing the existing api, orphaning existing roles saved in keychains
2. we're removing the "namespace" of an account name with a role name, creating the potential for conflicts if different accounts have the same role name.

Need to think a bit more about this, my first thought is maybe have an optional role-account argument on assume-role?

It doesn't seem like these changes are necessary for the console command.  Maybe splitting them up into two different PRs would help make things easier to review?

**Pull request closed without merging**